src/openrct2/platform/linux.c

```diff
@@ -78,36 +78,6 @@ void platform_get_exe_path(utf8 *outPath, size_t outSize)
     safe_strcpy(outPath, exePath, outSize);
 }
 
-/**
- * Default directory fallback is:
- *   - (command line argument)
- *   - $XDG_CONFIG_HOME/OpenRCT2
- *   - /home/[uid]/.config/OpenRCT2
- */
-void platform_posix_sub_user_data_path(char *buffer, size_t size, const char *homedir) {
-    const char *configdir = getenv("XDG_CONFIG_HOME");
```

Comment:
Here, this is what I mean: it used to check the environment variable first. And you removed that, not making this a "no change" PR.
(See my other comment)